lib/testers/rule-tester.js

```diff
@@ -129,114 +129,106 @@ function freezeDeeply(x) {
 // Public Interface
 //------------------------------------------------------------------------------
 
+// default separators for testing
+const DESCRIBE = Symbol("describe");
+const IT = Symbol("it");
+
 /**
- * Creates a new instance of RuleTester.
- * @param {Object} [testerConfig] Optional, extra configuration for the tester
- * @constructor
+ * This is `it` or `describe` if those don't exist.
+ * @this {Mocha}
+ * @param {string} text - The description of the test case.
+ * @param {Function} method - The logic of the test case.
+ * @returns {any} Returned value of `method`.
  */
-function RuleTester(testerConfig) {
+function defaultHandler(text, method) {
+    return method.apply(this);
+}
+
+class RuleTester {
 
     /**
-     * The configuration to use for this tester. Combination of the tester
-     * configuration and the default configuration.
-     * @type {Object}
+     * Creates a new instance of RuleTester.
+     * @param {Object} [testerConfig] Optional, extra configuration for the tester
+     * @constructor
      */
-    this.testerConfig = lodash.merge(
+    constructor(testerConfig) {
+
+        /**
+         * The configuration to use for this tester. Combination of the tester
+         * configuration and the default configuration.
+         * @type {Object}
+         */
+        this.testerConfig = lodash.merge(
+
+            // we have to clone because merge uses the first argument for recipient
```

Comment:
To avoid extra cloning, could we use empty object (or `Object.create(null)`) as first argument?